package.json

```diff
@@ -26,6 +26,7 @@
     "chalk": "^4.1.0",
     "discord.js-light": "^3.5.0",
     "express": "^4.17.1",
-    "mineflayer": "^3.0.0"
+    "mineflayer": "^4.3.0",
+    "yarn.lock": "^0.0.1-security"
```

Comment:
`yarn.lock` isn't a package, it's a file generated by yarn to make sure running `yarn` or `yarn install` installs all the right versions of packages. It is only installable with npm as the name is held to prevent malicious abuse.